### ISSUE TITLE
Octobox shall not start when SECRET_KEY_BASE is too small

### ISSUE THREAD

**Issue by NiR-** [octobox/octobox#1295]
Are you experiencing this issue in octobox.io or your own instance?
 - [ ] octobox.io
 - [x] My instance

🐞 **Problem**

When `SECRET_KEY_BASE` is too small the authentication works correctly until I got redirected from Github to Octobox: at this stage instead of seeing my inbox, I got redirected to the homepage like an unauthenticated user, but with no error message whatsoever.

In the logs I got this:

```
I, [2018-11-26T12:43:08.813593 #36]  INFO -- : [47dcbf09-6ec2-4e7a-b079-4f344c895734] method=GET path=/auth/github/callback format=html controller=SessionsController action=create status=500 error='ArgumentError: key must be 32 bytes or longer' duration=37.60 view=0.00 db=5.94
F, [2018-11-26T12:43:08.814987 #36] FATAL -- : [47dcbf09-6ec2-4e7a-b079-4f344c895734]   
F, [2018-11-26T12:43:08.815117 #36] FATAL -- : [47dcbf09-6ec2-4e7a-b079-4f344c895734] ArgumentError (key must be 32 bytes or longer):
F, [2018-11-26T12:43:08.815139 #36] FATAL -- : [47dcbf09-6ec2-4e7a-b079-4f344c895734]   
F, [2018-11-26T12:43:08.815166 #36] FATAL -- : [47dcbf09-6ec2-4e7a-b079-4f344c895734] app/models/user.rb:63:in `assign_from_auth_hash'
[47dcbf09-6ec2-4e7a-b079-4f344c895734] app/controllers/sessions_controller.rb:13:in `create'
```

(looks like the last line format is broken btw)

🎯 **Goal**

IMHO Octobox should break early (at startup) when the `SECRET_KEY_BASE` env var is too small because, inherently, an Octobox instance with broken authentication has no real purpose.

📋  **Steps to solve the problem**

I'm not a Ruby developer, so not sure where this should be done actually (`config/environments/production.rb`?).

**Comment by andrew** [comment]
@NiR- Agreed, it should definitely break early, here's probably the best place to add a check and `raise` an exception: https://github.com/octobox/octobox/blob/master/config/application.rb#L27

**Comment by andrew** [comment]
Fixed in #1300
